nixos/modules/services/security/bitwarden-directory-connector.nix

```diff
@@ -0,0 +1,316 @@
+{
+  config,
+  lib,
+  options,
```

Comment:
```suggestion
```
Sorry I left this in in my patch. It's not actually necessary the way I ultimately went about it.